sample_files/pre-commit-13.0/.pre-commit-config.yaml

```diff
@@ -31,6 +31,8 @@ repos:
   - id: check-docstring-first
   - id: check-executables-have-shebangs
   - id: check-merge-conflict
+    # exclude autogenerated files
+    exclude: /README\.rst$|\.pot?$
```

Comment:
this is the suggested solution (in the duplicate to the issue below) fwiw -- there isn't a way to discern between a stray marker and a rst header underline